app/views/drivers/index.html.slim

```diff
@@ -0,0 +1,24 @@
+table class="table"
+  thread
+    tr
+      th scope="col" Name
+      th scope="col" License Number
+      th scope="col" Mobile Number
+      th scope="col" Email Address
+      th scope="col" Age
+      th scope="col" Notes
+      th scope="col" Status
+      th scope="col" Actions
+  tbody
+    - @drivers.each do |driver|
+      tr
+        td = driver.name
+        td = driver.license_number
+        td = driver.mobile_number
+        td = driver.email_address
+        td = driver.age
+        td = driver.notes
+        td = driver.status
+        td
+          a(class="btn btn-primary" href=edit_driver_path(driver) role="button") Edit
+          a(class="btn btn-mini btn-danger" rel="nofollow" data-method="delete" data-confirm="Are you sure?" href=driver_path(driver)) Delete
```

Comment:
Rails has a [link_to method](https://www.rubyguides.com/2019/05/rails-link_to-method/) that we use to generate the html for links. Give it a try for these two links. The `delete` method will be a bit trickier. Let me know if you run into any problems

Reply:
we specifically want to keep these as styled buttons, there was some issues with styling the links like bootstrap buttons, we can have another crack at it though for sure